samples/cellular/nrf_cloud_multi_service/prj.conf

```diff
@@ -116,3 +116,4 @@ CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=n
 CONFIG_NRF_CLOUD_ALERT=y
 CONFIG_NRF_CLOUD_LOG_DIRECT=y
 CONFIG_NRF_CLOUD_LOG_OUTPUT_LEVEL=3
+CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS=n
```

Comment:
Does this mean by default it won't write these new things to the shadow?

Reply:
The application is adding the device status, so we do not want the library to also add it.

It's a workaround until I do this task: https://nordicsemi.atlassian.net/browse/IRIS-6792.